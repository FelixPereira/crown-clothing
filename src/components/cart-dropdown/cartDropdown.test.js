import React from 'react';
import { shallow } from 'enzyme';
import CartDropdown from './cart-dropdown';

it('render cart dropdown component', () => {
  expect(shallow(<CartDropdown />)).toMatchSnapshot();
});
