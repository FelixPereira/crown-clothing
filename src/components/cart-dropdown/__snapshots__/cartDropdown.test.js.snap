// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render cart dropdown component 1`] = `ShallowWrapper {}`;
